New documentation for the CTSM dust emission module (#3624)
ESCOMP/CTSM: doc/source/tech_note/Dust/CLM50_Tech_Note_Dust.rst

Dust Model
==============

Atmospheric dust is mobilized from the land by wind in the CLM. The most important factors determining soil erodibility and dust emission include the wind friction velocity, the vegetation cover, and the soil moisture. The latest CTSM allows users to choose between two dust emission schemes: One is Leung_2023 (:ref:`Leung et al. 2023<Leungetal2023>`; :ref:`Leung et al. 2024<Leungetal2024>`) which is the default for the CLM6 physics, and the other is Zender_2003 (:ref:`Mahowald et al. 2006<Mahowaldetal2006>`) based on the DEAD (Dust Entrainment and Deposition model of :ref:`Zender et al. (2003)<Zenderetal2003>`.

> One can control the use of the dust emission scheme by setting the namelist variable: 

Reviewer: @dmleung this is a detail on running that code that doesn't need to be here. I don't think we give the same instruction for other parameterizations.

``dust_emis_method = 'Zender_2003'``, which is the default for CLM5 physics or older.

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Leung et al. (2023) scheme (default for CLM6 physics or later):
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

Dust emission modeling is a threshold parameterization of an aeolian (wind-driven) process for both Leung_2023 and Zender_2003. For Leung_2023, in any given timestep the soil surface wind velocity :math:`u_{*s}` has to be greater than the fluid threshold friction velocity :math:`u_{*ft}` to generate saltation and dust emission:

.. math::
   :label: 30.1

   u_{*ft} = u_{*ft0}(D_{p},\rho_{a}) f_{m}(w)

where :math:`u_{*ft0}(D_{p},\rho_{a})` is the dry fluid threshold without the soil moisture effect :math:`f_{m}`, as a function of median soil diameter :math:`D_{p}` and air density :math:`\rho_{a}`. In CTSM for Leung_2023, :math:`D_{p}` is a globally uniform number of 130 :math:`\mu` m. :math:`u_{*ft0}(D_{p},\rho_{a})` is given by :ref:`Shao and Lu (2000)<ShaoLu2000>`:

.. math::
   :label: 30.2

   u_{*ft0}(D_{p},\rho_{a}) = \sqrt{\frac{A(\rho_{p} g D_{p} + \gamma / D_{p}) }{\rho_{a}} } 

> where :math:`g = 9.81` m s\ :sup:`-2` is gravity, :math:`\rho_{p} = 2650` kg m\ :sup:`-3` is typical soil particle density, and :math:`A = 0.0123` and :math:`\gamma = 1.65 \times 10^{-4}` kg s\ :sup:`-2` are empirical constants. 

Reviewer: There is a table of constants in the back Table 2.2.7, at least "g" is there. The "where" style is used in other places though, so the empirical constants should be handled that way. rho_{p} isn't anywhere else so can be here as well.

.. math::
   :label: 30.29

   F_{j} = F_d \sum _{i=1}^{I}M_{i,j}

where :math:`F_{j}` is the mass emission flux from the :math:`j` th aerosol bin. The bottom of this page shows how :math:`M_{i,j}` is calculated.

The current way of paritioning the emission fluxes before passing to the coupler is still being used, but the partition into the different bins is only required by BAM, not the current default MAM in CAM6 and CAM7. Therefore, for MAM, the four :math:`F_{j}` are summed up to become one total flux :math:`F_d` again. It is then redistributed to different individual modes of MAM following the Brittle Fragmentation Theory. The default CAM physics that is paired with the CLM6 physics is the CAM7 physics, which uses MAM4 (for minimal atmospheric chemistry) or MAM5 (for complex chemistry).

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
**Zender et al. (2003) scheme (default for CLM5 physics and older):**
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

The total vertical mass flux of dust, :math:`F_{j}` (kg m\ :sup:`-2` s\ :sup:`-1`), from the ground into transport bin :math:`j` is given by

> .. math::

Reviewer: @samsrabin I'm not sure that we could keep the old description of Zender. The CLM5.0 tech note gives that, so I think we should just add the new and remove the old bits about Zender.

Reviewer 2: If Zender isn't an option anymore, it should be removed, yes.

Dust Model
==============

Atmospheric dust is mobilized from the land by wind in the CLM. The most important factors determining soil erodibility and dust emission include the wind friction velocity, the vegetation cover, and the soil moisture. The latest CTSM allows users to choose between two dust emission schemes: One is Leung_2023 (:ref:`Leung et al. 2023<Leungetal2023>`; :ref:`Leung et al. 2024<Leungetal2024>`) which is the current default for the CLM6 physics, and the other is Zender_2003 (:ref:`Mahowald et al. 2006<Mahowaldetal2006>`) based on the DEAD (Dust Entrainment and Deposition model of :ref:`Zender et al. (2003)<Zenderetal2003>`, which is the default for the CLM5 or older physics.

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

> Leung et al. (2023) scheme (default for CLM6 physics or later):

Author: ```suggestion
Leung et al. (2023) scheme (default for CLM6 physics or later)
```

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

Dust emission modeling is a threshold parameterization of an aeolian (wind-driven) process for both Leung_2023 and Zender_2003. For Leung_2023, in any given timestep the soil surface wind velocity :math:`u_{*s}` has to be greater than the fluid threshold friction velocity :math:`u_{*ft}` to generate saltation and dust emission:

.. math::
   :label: 30.1

   u_{*ft} = u_{*ft0}(D_{p},\rho_{atm}) f_{m}(w)

where :math:`u_{*ft0}(D_{p},\rho_{atm})` is the dry fluid threshold without the soil moisture effect :math:`f_{m}`, as a function of median soil diameter :math:`D_{p}` and air density :math:`\rho_{atm}`. In CTSM for Leung_2023, :math:`D_{p}` is a globally uniform number of 130 :math:`\mu` m. :math:`u_{*ft0}(D_{p},\rho_{atm})` is given by :ref:`Shao and Lu (2000)<ShaoLu2000>`:

.. math::

>    :label: 30.2

Author: With mizuroute being added, now the dust chapter is 31 instead of 30. May want to change all the numbers to 31?

Reviewer: Yeah we should do that as a global thing when we get the chapter changes in.

Author: We removed all equation numbers and essentially changed those equation labels into some sort of alias.

   u_{*ft0}(D_{p},\rho_{atm}) = \sqrt{\frac{A(\rho_{p} g D_{p} + \gamma / D_{p}) }{\rho_{atm}} } 

where *g* is the acceleration of gravity (:numref:`Table Physical Constants`), :math:`\rho_{p} = 2650` kg m\ :sup:`-3` is typical soil particle density, and :math:`A = 0.0123` and :math:`\gamma = 1.65 \times 10^{-4}` kg s\ :sup:`-2` are empirical constants. 
The soil moisture effect :math:`f_{m}(w)` is a function of gravimetric soil moisture :math:`w` (kg water / kg soil) at the topmost soil layer. 

:math:`w` is converted from the CLM volumetric soil moisture :math:`\theta` and porosity (saturation moisture :math:`\theta_{sat}`) at the topmost soil layer:

.. math::
   :label: 30.3

>    w=\theta\frac{ \rho _{water} }{\rho_{bulk} }

Author: ```suggestion
   w=\theta\frac{ \rho _{liq} }{\rho_{bulk} }
```

> Note that :math:`w` in CTSM is conventionally (since CLM3; :ref:`Mahowald et al. 2006<Mahowaldetal2006>`) treated as a sum of both liquid and ice/frozen soil moisture at the topmost soil layer, i.e., :math:`w = w_{liq,1} + w_{ice,1}`. :math:`\rho_{liq}` is typical liquid water density (:numref:`Table Physical Constants`), and bulk density :math:`\rho_{bulk}` is given by 

Author: This should be written as
"Note that w and theta
 in CTSM ARE conventionally ..."

Reviewer: Sounds like a good change to make.

Author: Done.

.. math::
   :label: 30.4

   \rho_{bulk} = (1 - \theta_{sat} ) \rho_{p}

Then, the soil moisture effect :math:`f_{m}(w)` on increasing the fluid threshold is given by :ref:`Fecan et al. (1999)<Fecanetal1999>`:

.. math::
   :label: 30.5

   f_{m}(w) =\left\{\begin{array}{l} {1{\rm \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; \; for\; }w\le w_{t} } \\ {\sqrt{1+1.21\left[100\left(w-w_{t} \right)\right]^{0.68} } {\rm \; \; for\; }w>w_{t} } \end{array}\right.

where :math:`w_{t}` increases with clay fraction :math:`f_{clay}`:

.. math::
   :label: 30.6

   w_{t} =0.01a\left(17f_{clay} +14f_{clay}^{2} \right){\rm \; \; \; \; \; \; 0}\le f_{clay} =\% clay\times 0.01\le 1

> where :math:`a=f_{clay}^{-1}` for tuning purposes. Note that this is different from the paper in which a = 1 was chosen. The coefficient 0.01 is used for converting :math:`w_{t}` from % to fraction  (kg water / kg soil), :math:`\theta` is the volumetric soil moisture (water+ice) in the topmost soil layer (m\ :sup:`-3`\ water \ m\ :sup:`-3` soil) (section :numref:`Soil Water`), :math:`\rho _{liq}` is the density of liquid water (kg m\ :sup:`-3`) (:numref:`Table Physical constants`), and :math:`\rho _{bulk}` is the bulk density of soil in the top soil layer (kg m\ :sup:`-3`) defined as in section :numref:`Soil and Snow Thermal Properties` rather than as in :ref:`Zender et al. (2003)<Zenderetal2003>`. :math:`f_{clay}` is the mass fraction of clay particles in the topmost soil layer and %clay comes from the surface dataset (section :numref:`Surface Data`).

Author: After
"Note that this is different from the paper",
cite Leung 2023 and Leung 2024 here.

Author: Also, some of the definitions of constants like ρ_liq and ρ_bulk should be moved to the previous lines when the equations are first shown.

Reviewer: Yes good things to do here.

Author: Done.

> Another essential dust emission threshold is the impact / dynamic threshold :math:`u_{*it}`, which is the lowest friction velocity or wind stress to matintain saltation:

Author: ```suggestion
Another essential dust emission threshold is the impact/dynamic threshold :math:`u_{*it}`, which is the lowest friction velocity or wind stress to matintain saltation:
```

.. math::
   :label: 30.7

   u_{*it} = B_{*it} u_{*ft0}

where :math:`B_{*it}` is a constant on Earth following :ref:`Kok et al. (2012)<Koketal2012>`. The above equations imply that :math:`u_{*it} \, < \, u_{*ft0} \le \, u_{*ft}`. This means that the winds need a bigger momentum to initiate saltation and dust emission but can reduce below :math:`u_{*ft}` and still maintain a weak dust emission flux. The emission flux goes to zero when :math:`u_{*s}` drops below :math:`u_{*it}`.

> The total vertical mass emission flux of dust, :math:`F_{d}` (kg m\ :sup:`-2` s\ :sup:`-1`), from the ground into a transport mode/bin :math:`j` of aerosol is given by :ref:`Kok et al. (2014a)<Koketal2014a>`:

Author: ```suggestion
The total vertical mass emission flux of dust, :math:`F_{d}` (kg m\ :sup:`-2` s\ :sup:`-1`), from the ground into a transport mode/bin :math:`j` of aerosol is based on :ref:`Kok et al. (2014a)<Koketal2014a>`:
```

.. math::
   :label: 30.8

>    F_{d} = \eta C_{tune} C_{d} f_{bare} f_{clay} \frac{ \rho_{atm} (u^2_{*s} - u^2_{it} ) }{ u^2_{it} }  \left( \frac{ u^2_{*s} }{u^2_{it} } \right) ^\kappa

Author: This is not really f_clay, it is f_clay' and it needs a description here.
Look at line 700 of DustEmisLeung2023.F90:
https://github.com/ESCOMP/CTSM/blob/b4b-dev/src/biogeochem/DustEmisLeung2023.F90#L700

Author: Note that the calculation of this term is in line 822 of biogeophys/SoilStateInitTimeConstMod.F90:
https://github.com/ESCOMP/CTSM/blob/b4b-dev/src/biogeophys/SoilStateInitTimeConstMod.F90#L822

Reviewer: Good things to do.

Author: Done.

where :math:`C_{tune} = 0.05` is a constant, and :math:`F_{d}` is the total emission flux summed across modes/bins following a revised form of :ref:`Kok et al. (2014b)<Koketal2014b>`. The dust emission flux goes to zero when :math:`u_{*s} \, < \, u_{*it}`. :math:`\rho_{atm}` is surface air density from CAM (the atm model). 
:math:`\kappa` is the fragmentation exponent, and :math:`C_{d}` is the dust emission coefficient (or the soil erodibility coefficient):

.. math::
   :label: 30.9

   C_{d} = C_{d0} \exp{ (-C_{e} \frac{ u_{*st} - u_{st0} }{ u_{st0} } ) }

.. math::
   :label: 30.10

   \kappa = C_{\kappa} \frac{ u_{*st} - u_{st0} }{ u_{st0} }

> where :math:`C_{\kappa} = 2.7`, :math:`u_{st0} = 0.16` m s:sup:`-3`, :math:`C_{d0} = 4.4 \times 10^{-5}`, and :math:`C_{e} = 2.0`. :math:`F_{d}` thus roughly scales with :math:`u^{2+\kappa}_{*s}`, where :math:`\kappa \sim 1` over major deserts and :math:`\sim 3` or higher over semiarid and nonarid regions. Since :ref:`Kok et al. (2014a)<Koketal2014a>` has not measured :math:`\kappa > 3` in their measurements, we cap :math:`\kappa` at a maximum value (currently set as 2.5). :math:`u_{*st}` is the standardized wet fluid threshold at a typical atmospheric surface air density (Kok et al., 2014):

Author: ```suggestion
where :math:`C_{\kappa} = 2.7`, :math:`u_{st0} = 0.16` m s :sup:`-3`, :math:`C_{d0} = 4.4 \times 10^{-5}`, and :math:`C_{e} = 2.0`. :math:`F_{d}` thus roughly scales with :math:`u^{2+\kappa}_{*s}`, where :math:`\kappa \sim 1` over major deserts and :math:`\sim 3` or higher over semiarid and nonarid regions. Since :ref:`Kok et al. (2014a)<Koketal2014a>` has not measured :math:`\kappa > 3` in their measurements, we cap :math:`\kappa` at a maximum value (currently set as 2.5). :math:`u_{*st}` is the standardized wet fluid threshold at a typical atmospheric surface air density (Kok et al., 2014):
```

.. math::
   :label: 30.11

   u_{*st} = u_{*ft} \sqrt{ \rho_{atm} / \rho_{0atm}}

> where :math:`\rho_{0a} = 1.225` kg m\ :sup:`-3`. As can be seen, :math:`u_{*st}` scales with :math:`u_{*ft}` and thus soil moisture :math:`w`. Therefore, moisture :math:`w` decreases soil erodibility :math:`C_{d}` but increases dust emission sensitivity :math:`\kappa` to the winds.

Author: ```suggestion
where :math:`\rho_{0atm} = 1.225` kg m\ :sup:`-3`. As can be seen, :math:`u_{*st}` scales with :math:`u_{*ft}` and thus soil moisture :math:`w`. Therefore, moisture :math:`w` decreases soil erodibility :math:`C_{d}` but increases dust emission sensitivity :math:`\kappa` to the winds.
```

> :ref:`Kok et al. (2014a)<Koketal2014a>` is different from many other dust emission parameterizations in the way that the soil erodibility :math:`C_{d}` is not a time-invariant input data but is a transient function, with erodibility increasing with reducing :math:`u^2_{*ft}` (and thus implicitly soil moisture). Similarly, the fragmentation exponent :math:`\kappa` is also transient but increases with enhancing soil moisture.

Author: ```suggestion
:ref:`Kok et al. (2014a)<Koketal2014a>` is different from many other dust emission parameterizations in the way that the soil erodibility :math:`C_{d}` is not a time-invariant input data but is a transient function, with erodibility increasing with reducing :math:`u^2_{*ft}` (and thus implicitly soil moisture). Similarly, the fragmentation exponent :math:`\kappa` is also transient and increases with enhancing soil moisture.
```

The grid cell fraction of exposed bare soil suitable for dust mobilization :math:`f_{bare}` is given by

.. math::
   :label: 30.12

   f_{bare} =\left(1-f_{lake} \right)\left(1-f_{sno} \right)\left(1-f_{v} \right)\frac{w_{liq,1} }{w_{liq,1} +w_{ice,1} }

where :math:`f_{lake}` and :math:`f_{sno}` are the CLM grid cell fractions of lake (section :numref:`Surface Data`) and snow cover (section :numref:`Snow Covered Area Fraction`), all ranging from zero to one. Not mentioned by :ref:`Zender et al. (2003)<Zenderetal2003>`, :math:`w_{liq,\, 1}` and :math:`w_{ice,\, 1}` are the CLM top soil layer liquid water and ice contents (mm) entered as a ratio expressing the decreasing ability of dust to mobilize from increasingly frozen soil. The grid cell fraction of vegetation cover,\ :math:`f_{v}`, is defined as

.. math::
   :label: 30.13

   0\le f_{v} =\frac{\mathrm{VAI}}{\mathrm{VAI_{thr}} } \le 1{\rm \; \; \; \; where\; } \mathrm{VAI_{thr}} =0.6{\rm \; m}^{2} {\rm m}^{-2}

> where :math:`\mathrm{VAI}=\mathrm{LAI}+\mathrm{SAI}` is the vegetation area index as a sum of the CLM leaf and stem area index values (m :sup:`2` m\ :sup:`-2`) averaged at the land unit level so as to include all the pfts and the bare ground present in a vegetated land unit. Currently, Leung_2023 in CTSM sets the areas with :math:`\mathrm{VAI_{thr}}` smaller than or equal to 0.6 m:sup:`2` m\ :sup:`-2` to be dust-emitting grids, different from the :ref:`Leung et al. (2024)<Leungetal2024>` paper that set :math:`\mathrm{VAI_{thr}}` to be 1 m:sup:`2` m\ :sup:`-2`. :math:`\mathrm{LAI}` and :math:`\mathrm{SAI}` may be prescribed from the CLM input data (section :numref:`Phenology and vegetation burial by snow`) or simulated by the CLM biogeochemistry model (section :numref:`rst_Vegetation Phenology and Turnover`).

Author: ```suggestion
where :math:`\mathrm{VAI}=\mathrm{LAI}+\mathrm{SAI}` is the vegetation area index as a sum of the CLM leaf and stem area index values (m :sup:`2` leaf m\ :sup:`-2` grid) averaged at the land unit level so as to include all the pfts and the bare ground present in a vegetated land unit. Currently, Leung_2023 in CTSM sets the areas with :math:`\mathrm{VAI_{thr}}` smaller than or equal to 0.6 m :sup:`2` m\ :sup:`-2` to be dust-emitting grids, different from the :ref:`Leung et al. (2024)<Leungetal2024>` paper that set :math:`\mathrm{VAI_{thr}}` to be 1 m :sup:`2` m\ :sup:`-2`. :math:`\mathrm{LAI}` and :math:`\mathrm{SAI}` may be prescribed from the CLM input data (section :numref:`Phenology and vegetation burial by snow`) or simulated by the CLM biogeochemistry model (section :numref:`rst_Vegetation Phenology and Turnover`).
```

Author: Need the CLM people to confirm if the last sentence of this is correct.

.. math::
   :label: 30.15

    F_{eff} = f_{rock}

And for a patch/PFT with short vegetation (shrub, grass, crop) we use:

.. math::
   :label: 30.16

    F_{eff} = f_{veg} 

The rock drag partition factor scales with the surface roughness density of rocks, captured by the aeolian roughness length :math:`z_{0a}` from the satellite-derived dataset from :ref:`Prigent et al. (2005)<Prigentetal2005>`. The expression was first developed by :ref:`Marticorena and Bergametti (1995)<MarticorenaBergametti1995>`, which is more valid for the low-roughness surfaces:

.. math::
   :label: 30.17

>    f_{rock} = 1 - \frac{\ln(\frac{z_{0a}}{z_{0s}})}{\ln(b_{1}[\frac{X}{z_{0s}}]^{b_{2}})}

Author: ```suggestion
   f_{rock} = 1 - \frac{\ln(\frac{z_{0a}}{z_{0s}})}{\ln[b_{1}(\frac{X}{z_{0s}})^{b_{2}}]}
```

> where :math:`X = 10 m` is the distance downstream the point of discontinuity in surface obstacle, :math:`b_{1} = 0.7` and :math:`b_{2} = 0.8` are coefficients (:ref:`Darmenova et al., 2009<Darmenovaetal2009>`), :math:`z_{0s}` is the soil roughness length. :math:`z_{0a}` is from :ref:`Prigent et al. (2005)<Prigentetal2005>` and should not be confused with the aerodynamic roughness length :math:`Z_{0}` from the model. This equation only applies for gridcells with LAI smaller than the LAI threshold for dust emission.

Author: Maybe this is better

```suggestion
where :math:`X = 10` m is the distance downstream the point of discontinuity in surface obstacle, :math:`b_{1} = 0.7` and :math:`b_{2} = 0.8` are coefficients (:ref:`Darmenova et al., 2009<Darmenovaetal2009>`), :math:`z_{0s}` is the soil roughness length. :math:`z_{0a}` is from :ref:`Prigent et al. (2005)<Prigentetal2005>` and should not be confused with the aerodynamic roughness length :math:`Z_{0}` from the model. This equation only applies for gridcells with VAI smaller than the VAI threshold for dust emission.
```

Reviewer: The formatting change is subtle, but the correction to LAI is critical.

> where :math:`X = 10` m is the distance downstream the point of discontinuity in surface obstacle, :math:`b_{1} = 0.7` and :math:`b_{2} = 0.8` are coefficients (:ref:`Darmenova et al., 2009<Darmenovaetal2009>`), :math:`z_{0s}` is the soil roughness length. :math:`z_{0a}` is from :ref:`Prigent et al. (2005)<Prigentetal2005>` and should not be confused with the aerodynamic roughness length :math:`Z_{0}` from the model. This equation only applies for gridcells with VAI smaller than the VAI threshold for dust emission.

Reviewer: Some more uses of VAI to fix. Here and below.

Author: I think VAI is correct? I was changing most of the usage of LAI to VAI.

Reviewer: Oh, VAI is LAI+SAI. OK, that makes sense. Sorry thought this was a typo.

> The vegetation drag partition factor scales with vegetation density as captured by LAI following :ref:`Okin (2008)<Okin2008>` and :ref:`Pierre et al. (2014)<Pierreetal2014>`:

Author: ```suggestion
The vegetation drag partition factor scales with vegetation density as captured by VAI following :ref:`Okin (2008)<Okin2008>` and :ref:`Pierre et al. (2014)<Pierreetal2014>`:
```

.. math::
   :label: 30.18

   f_{veg} = \frac{K + f_{0} c }{K + c}

.. math::
   :label: 30.19

   K = 2 (\frac{1}{f_{v}} - 1) = 2 (\frac{1}{\mathrm{VAI}/\mathrm{VAI_{thr}}} - 1)

> where :math:`f_{0} = 0.33` is the wind dissipation right behind the obstacle, :math:`c = 4.8 m` is the average e-folding distance for wind momentum to recover after dissipated by the plant, and :math:`K` is defined as the average normalized gap length between two obstacles given the vegetation density quantified by VAI. This equation assumes that the gap length K goes to zero as VAI increases and approaches :math:`\mathrm{VAI_{thr}}`.

Author: ```suggestion
where :math:`f_{0} = 0.33` is the wind dissipation right behind the obstacle, :math:`c = 4.8` m is the average e-folding distance for wind momentum to recover after dissipated by the plant, and :math:`K` is defined as the average normalized gap length between two obstacles given the vegetation density quantified by VAI. This equation assumes that the gap length K goes to zero as VAI increases and approaches :math:`\mathrm{VAI_{thr}}`.
```

> Then, the fraction of time within a timestep :math:`\Delta t` when dust emission is active is accounted for by Leung_2023, known as the intermittency factor :math:`\eta`. The fraction of time :math:`\eta \in [0,1]` is due to the boundary-layer turbulence that causes sub-timestep turbulent wind fluctuations that cross the dust emission thresholds multiple times within a timestep, in turn initiating and shutting off emission at time within :math:`\Delta t`. The parameterization of :math:`\eta` comes from :ref:`Comola et al. (2019)<Comolaetal2019>`, in which a statistical substepping method was proposed to account for the temporary shutoff of dust emission fluxes. 

Author: ```suggestion
Then, the fraction of time within a timestep :math:`\Delta t` (typically of 30 minutes for a ~1° horizontal grid) when dust emission is active is accounted for by Leung_2023, known as the intermittency factor :math:`\eta`. The fraction of time :math:`\eta \in [0,1]` is due to the boundary-layer turbulence that causes sub-timestep turbulent wind fluctuations that cross the dust emission thresholds multiple times within a timestep, in turn initiating and shutting off emission at time within :math:`\Delta t`. The parameterization of :math:`\eta` comes from :ref:`Comola et al. (2019)<Comolaetal2019>`, in which a statistical substepping method was proposed to account for the temporary shutoff of dust emission fluxes. 
```

The fraction of time :math:`\eta` is parameterized using the surface winds and thresholds at the saltation height. Therefore, the friction velocities are translated using the log law of the wall to the saltation height, which was defined as :math:`z_{sal}` = 0.1 m by :ref:`Comola et al. (2019)<Comolaetal2019>`:

.. math::
   :label: 30.20

>    u_{s} =  \frac{u_{*s}}{k} \ln(z_{sal}/z_{0a})

Author: Need to mention von Karman constant here and refer to table 2.2.7.

Reviewer 3: Side-note for this and other equations:
As you noticed, equation numbers change automatically as the documentation evolves. You are better off replacing the `:label:` text with a short description to avoid the impossible task of keeping these numbers up-to-date.

Author: Hi Sam @slevis-lmwg I am not good at rst and am not sure what you mean by replacing the `:label:` with a short description. do you have any exmaple in other rst files?

Reviewer 3: There's an example in the third bullet of the latest documentation style guide:
https://escomp.github.io/CTSM//users_guide/working-with-documentation/docs-style-guide.html#documentation-style-guide

Author: Done. I converted the equation numbers in the rst file into some alias.

.. math::
   :label: 30.21

   u_{ft} =  \frac{u_{*ft}}{k} \ln(z_{sal}/z_{0a})

.. math::
   :label: 30.22

   u_{it} =  \frac{u_{*it}}{k} \ln(z_{sal}/z_{0a})

> With saltation-height variables defined, the instantaneous wind :math:`\tilde{u}_s` is assumed by Comola to follow a Gaussian distribution with a mean equal to the mean wind speed and the spread :math:`\sigma_{u_{s}}`  parameterized by the Similarity Theory (:ref:`Panofsky et al., 1977<Panofskyetal1977>`):

Author: ```suggestion
where :math:`z_{0a}`, the aeolian roughness length, is set to be 10 :sup:`-4` m here for simplicity. With saltation-height variables defined, the instantaneous wind :math:`\tilde{u}_s` is assumed by Comola to follow a Gaussian distribution with a mean equal to the mean wind speed and the spread :math:`\sigma_{u_{s}}`  parameterized by the Similarity Theory (:ref:`Panofsky et al., 1977<Panofskyetal1977>`):
```

.. math::
   :label: 30.23

   \tilde{u}_s \sim N(u_s, \sigma_{u_s}) 

And the fluctuation strength is parameterized by the similarity theory:

.. math::
   :label: 30.24

   \sigma_{u_s} = u_{*s} \left( 12 - 0.5 \frac{z_i}{L} \right)^{1/3}

>    \quad \text{for } \left( 12 - 0.5 \frac{z_i}{L} \right) \ge 0

Author: ```suggestion
   \quad \text{for } \left 12 - 0.5 \frac{z_i}{L} \right \ge 0
```

> where :math:`z_i = 1000` m is the planetary boundary-layer height set as a constant for now, and :math:`L` is the Obukhov length. This means the instantaneous wind's fluctuation comes from both a shear contribution and a buoyancy contribution. 

Author: ```suggestion
where :math:`z_i = 1000` m is the planetary boundary-layer height set as a constant for now, and :math:`L` is the Obukhov length scale. This means the instantaneous wind's fluctuation comes from both a shear contribution and a buoyancy contribution. 
```

Then, the total fraction of time :math:`\eta` when saltation is active within a model timestep is then formulated as

.. math::
   :label: 30.25

   \eta = 1 - P_{ft} + \alpha \left( P_{ft} - P_{it} \right)

> where :math:`P_{it}` is the cumulative probability that the instantaneous wind :math:`\tilde{u}_s` does not exceed the impact threshold :math:`u_{it}`, and :math:`P_{ft}` is the cumulative probability that :math:`\tilde{u}_s` does not exceed the fluid threshold :math:`u_{ft}`. The fluid threshold crossing fraction :math:`\alpha` is defined as the rate of :math:`\tilde{u}_s` sweeping across :math:`u_{ft}` divided by the rate of sweeping across :math:`u_{it}` and :math:`u_{ft}` summed up. The detailed explanation of this formula is in :ref:`Leung et al. (2023)<Leungetal2023>`. Here we just document the equations for each term:

Author: ```suggestion
where :math:`P_{it}` is the cumulative probability that the instantaneous wind :math:`\tilde{u}_s` does not exceed the impact threshold :math:`u_{it}`, and :math:`P_{ft}` is the cumulative probability that :math:`\tilde{u}_s` does not exceed the fluid threshold :math:`u_{ft}`. The fluid threshold crossing fraction :math:`\alpha` is defined as the rate of :math:`\tilde{u}_s` sweeping across :math:`u_{ft}` divided by the rate of sweeping across :math:`u_{it}` and :math:`u_{ft}` summed up. The detailed physical interpretation of this formula is in :ref:`Leung et al. (2023)<Leungetal2023>`. Here we just document the equations for each term:
```

.. math::
   :label: 30.26

   \alpha \approx \left\{ \exp\left[ 
      \frac{u_{ft}^2 - u_{it}^2 - 2u_s(u_{ft}-u_{it})}{2 \sigma^2_{u_s} }
      \right] + 1 \right\}^{-1}

Then, the fraction of time in :math:`\Delta t` when :math:`\tilde{u}_s` is above :math:`u_{ft}` is given by :math:`1 - P_{ft}`, where

.. math::
   :label: 30.27

   P_{ft} = \frac{1}{2} \left[ 1 + \operatorname{erf}
   \left( \frac{u_{ft} - u_s}{\sqrt{2} \sigma_{u_s}} \right) \right]

And, the fraction of time in :math:`\Delta t` when :math:`\tilde{u}_s` is dropping from above :math:`u_{ft}` to within the hysteresis zone between :math:`u_{it}` and :math:`u_{it}` is given by :math:`\alpha \left( P_{ft} - P_{it} \right)`, where
 
.. math::
   :label: 30.28

   P_{it} = \frac{1}{2} \left[ 1 + \operatorname{erf}
   \left( \frac{u_{it} - u_s}{\sqrt{2} \sigma_{u_s}} \right) \right]

And so the fraction of time :math:`\eta` within :math:`\Delta t` with active emission is determined for :eq:`30.8`.

> The total vertical mass emission flux of dust, :math:`F_{d}` (kg m\ :sup:`-2` s\ :sup:`-1`) is then computed with all the above equations and terms. The emission flux is then passed through the coupler to the atmospheric model (CAM) to simulate dust aerosol transport and deposition. The default aerosol model supported in the CAM6 and CAM7 physics is the Modal Aerosol Model (MAM). CAM7 uses the 5-mode MAM (MAM5), in which three modes (Aitken, accumulation, and coarse modes) by default contain dust. The total mass emission flux per grid is partitioned into the three modes following the Brittle Fragmentation Theory (BFT) by Kok et al. (2014) and later modified by Meng et al. (2022). In the current model version, the fractions of dust emission flux partitioned in the three modes are 1.65E-05, 0.021, and 0.979 for the Aitken (0.01–0.1 um), accumulation (0.1–1 um), and coarse (1–10 um) modes, respectively. These values are prescribed in the MAM code inside CAM.

Author: Need to add links to the bibliography for the papers here.

Author: Done.

In the early CESM versions, CAM employed the Bulk Aerosol Model (BAM) as the default aerosol model. Thus, the emission fluxes in CTSM is by default partitioned into 4 bins before passing to the coupler:

.. math::
   :label: 30.29

   F_{j} = F_d \sum _{i=1}^{I}M_{i,j}

where :math:`F_{j}` is the mass emission flux from the :math:`j` th aerosol bin.

> The current way of paritioning the emission fluxes before passing to the coupler is still being used, but the partition into the different bins is only required by BAM, not the current default MAM in CAM6 and CAM7. Therefore, for MAM, the four :math:`F_{j}` are summed up to become one total flux :math:`F_d` again. It is then redistributed to different individual modes of MAM following the Brittle Fragmentation Theory. The default CAM physics that is paired with the CLM6 physics is the CAM7 physics, which uses MAM4 (4 modes) or MAM5 (5 modes).

Author: ```suggestion
The current way of paritioning the emission fluxes before passing to the coupler is still being used, but the partition into the different bins is only required by BAM, not the current default MAM in CAM6 and CAM7. Therefore, for MAM, the four :math:`F_{j}` are summed up to become one total flux :math:`F_d` again inside MAM. It is then redistributed to different individual modes of MAM following the three values mentioned above from the BFT. 
```

> In equation :eq:`30.29` we sum :math:`M_{i,\, j}` over :math:`I=3` source modes :math:`i` where :math:`M_{i,\, j}` is the mass fraction of each source mode :math:`i` carried in each of *:math:`J=4`* transport bins :math:`j`

Author: ```suggestion
In equation :eq:`30.29` we sum :math:`M_{i,\, j}` over :math:`I=3` source modes :math:`i` where :math:`M_{i,\, j}` is the mass fraction of each source mode :math:`i` carried in each of :math:`J=4` transport bins :math:`j`
```
